Prac05/word_occurences.py

```diff
@@ -0,0 +1,22 @@
+"""
+CP1404/CP5632 Practical
+User strings with dictionary printing.
+Instruction time: 8:32 p.m.
+Complete time: 9:17 p.m.
+Estimate time: 40 minutes
+Actual time: 45 minutes
+"""
+word_occurrences = {}
+user_input = input("Your text:").lower()
+glossarys = user_input.split()  # by this time I need to confirm the splitting context it is match.
+
+for glossary in glossarys:
```

Comment:
maybe glossary is not the best name for list of random words; maybe e.g. "row_elements" woudl be better